Christina Kopecky

ckopecky opened the pull request.

ckopecky committed:
connected mongoose to server

created friendSchema

created friendController for API endpoints

utilized CRUD methods to create API

ckopecky committed:
took out typo - no dot on line 84 of friend Controller

ckopecky committed:
inserted some error handlers

will finish tomorrow

designerexpert commented on friendModel.js:
```diff
@@ -0,0 +1,33 @@
+        type: String,  // String, required
+        required: true
+    },
+    age: {
```
For this portion consider using a custom validation for the age to be between 1 to 120.
Reference this document: http://mongoosejs.com/docs/validation.html#custom-validators

designerexpert commented:
# Comments
* Good progress so far. I specially like the way you separate your routers from your callbacks. Consider using a separate file for each. One for Routes and One for the Route Controllers, keep things even clearer.

ckopecky committed:
oh goodness forgot to push today

created front-end for project

header page

on click of button, it shows friendslist component

ckopecky committed:
create friend constructed

just need to make it functional

ckopecky committed:
got delete to work by putting a blue close icon in the upper

right hand corner of card and writing a handleDelete func
